boards/nxp/mcx_nxxx_evk/mcx_n5xx_evk_mcxn547_cpu0.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: mcx_n5xx_evk/mcxn947/cpu0
+name: NXP MCX-N5XX-EVK (CPU0)
+type: mcu
+arch: arm
+ram: 320
+flash: 2048
+toolchain:
+  - zephyr
+  - gnuarmemb
+supported:
+  - adc
```

Comment:
I would not suggest to enable all features without checking the corresponding tests.